lib/main.js

```diff
@@ -1,9 +1,23 @@
 /*!
  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
-
+import bedrock from 'bedrock';
+const {config} = bedrock;
 import './config.js';
-export {OAuth2Client} from './oauth2-client.js';
+import {getAuthzClients} from './oauth2-client.js';
 import * as storage from './storage.js';
 export {storage};
 export * from './errorCheck.js';
+export * from './authzHttpClient.js';
+export * from './oauth2-client.js';
+
+bedrock.events.on('bedrock.ready', async () => {
+  const promises = [];
+  for(const namespace in config) {
+    if(config[namespace].authorization) {
+      const clients = config[namespace].authorization;
+      promises.push(getAuthzClients({clients}));
```

Comment:
Do we still intend to process.exit() on non-retryable errors?

Reply:
Yes, the exit occurs here: https://github.com/digitalbazaar/bedrock-oauth2-client/blob/main/lib/oauth2-client.js#L184